openreview/tools.py

```diff
@@ -91,7 +90,6 @@ def build_groups(conference_group_id, default_params=None):
 
     return sorted(groups.values(), key=lambda x: len(x.id))
 
-@deprecated
```

Comment:
are we giving up on this for now?

Reply:
Yes, it means to add a new library to the setup.py, I don't know if it is worth. When we have more deprecated functions we could add it.